test/auth.controller.e2e-spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Test, TestingModule} from "@nestjs/testing"
+import { HttpStatus, INestApplication} from "@nestjs/common"
+import * as request from "supertest"
+import { AppModule } from "../src/app.module"
+import { UserService } from "../src/user/user.service"
+import { UserModel } from "../src/user/user.model"
+
+
+describe('AuthController (e2e)',()=>{
+    let app :INestApplication
+    let userService: UserService
+
+    beforeEach(async()=>{
+        await userService.deleteAll()
+    })
+    afterAll(async()=>{
+        await userService.deleteAll()
+    })
+
+    beforeAll(async ()=>{
+        const moduleFixture: TestingModule = await Test.createTestingModule({
+            imports:[AppModule],
+        }).compile()
+
+        app =  moduleFixture.createNestApplication()
+        userService = moduleFixture.get<UserService>(UserService)
+        await app.init()
+    })
+    afterAll(async ()=>{
+        await app.close()
+    })
+
+    describe('When trying login',()=>{
```

Comment:
change  `When trying login` to `When trying login in `

Reply:
++